File: src/org/pcsoft/plugins/intellij/inno_setup/script/others/IssCommenter.java
package org.pcsoft.plugins.intellij.inno_setup.script.others;

import com.intellij.lang.Commenter;
import org.jetbrains.annotations.Nullable;

/**
 * Created by Christoph on 23.12.2014.
 */
public class IssCommenter implements Commenter {

    @Nullable
    @Override
    public String getLineCommentPrefix() {
        return ";";
    }

    @Nullable
    @Override
    public String getBlockCommentPrefix() {
        return null;
    }

    @Nullable
    @Override
    public String getBlockCommentSuffix() {
        return null;
    }

    @Nullable
    @Override
    public String getCommentedBlockCommentPrefix() {
        return null;
    }

    @Nullable
    @Override
    public String getCommentedBlockCommentSuffix() {
        return null;
    }
}

File: src/main/kotlin/org/pcsoft/intellij/plugin/inno_setup/language/IssCommenter.kt
package org.pcsoft.intellij.plugin.inno_setup.language

import com.intellij.lang.Commenter

class IssCommenter : Commenter {
    override fun getLineCommentPrefix(): String = ";"
    override fun getBlockCommentPrefix(): String? = null
    override fun getBlockCommentSuffix(): String? = null
    override fun getCommentedBlockCommentPrefix(): String? = null
    override fun getCommentedBlockCommentSuffix(): String? = null
}
